### ISSUE TITLE
508-defect-2: Headings should be logically ordered on view my payments page

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#50632]
### Point of contact

@joshkimux

### Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Details

Currently, the visual hierarchy of the view my payments page and the semantic hierarchy of headings of the view my payments page are unaligned.

<img width="1624" alt="Screen Shot 2022-12-12 at 3 44 52 PM" src="https://user-images.githubusercontent.com/14154792/207151552-4ad12dde-1684-488e-b8e0-5a43e0f76dc6.png">

### Visually, it looks like this:

2. Payments returned
  3. What if I find a check that I reported missing?
2. What if I need to change my direct deposit or contact information?
2. What if I’m missing a payment?
  3. Need help?

### Semantically (how a screen reader user may hear this page)

2. Payments returned
  3. What if I find a check that I reported missing?
  3. What if I need to change my direct deposit or contact information?
  3. What if I’m missing a payment?
  3. Need help?

### Reproduction steps

1. Log in as user 228 and [log into view my payments page](https://staging.va.gov/va-payment-history/payments/)
2. Using [headingsmap](https://chrome.google.com/webstore/detail/headingsmap/flbjommegcjonpdmenkdiocclhjacmbi?hl=en), confirm that the visual and semantic hierarchy are unaligned

### Two steps are needed to close out this ticket:
- [ ] Determine the intended hierarchy of the page
- [ ] Fix the headings match to match that intended hierarchy

### Example suggestion
Looking at the design, we may want to consider adding an additional heading to contextualize the questions. For example, adding a "frequently asked questions" `h2` above the `h3`s can result in a fix that will only require removing the utility classes that have resized the `h3`s into `h2`s.

<img width="1118" alt="Screen Shot 2022-12-12 at 3 55 57 PM" src="https://user-images.githubusercontent.com/14154792/207152710-79e967ff-5cc0-47f4-a6eb-7ee98721f183.png">

### References, articles, or WCAG support

1.
2.
3.
...

### Type of issue

- [X] Screenreader
- [ ] Keyboard
- [ ] Focus
- [X] Headings
- [ ] Color
- [ ] Zoom
- [X] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by jacobworrell** [comment]
@joshkimux we've added a solution approach section to your ticket making clear how we intend to proceed. 

"Solution approach
We are going to take our cues from the approved designs and make the semantics match what is presented on the page."

**Comment by jaredcunha** [comment]
Looking more at the design, I think all the headings should be a Level 2, including:
- "What if I find a check that I reported missing?"
- "Need help?"

While I like the suggestion of using Frequently Asked Questions, I don't see other examples of that elsewhere (looked at about 7 or so pages). Usually, these questions simply follow the rest of the content.

So content hierarchy, both semantically and visually, will be: 

```
1. Your VA payments
2. Payments you received
2. Payments returned
2. What if I find a check that I reported missing?
2. What if I need to change my direct deposit or contact information?
2. What if I’m missing a payment?
2. Need help?

```

**Comment by jstrothman** [comment]
@joshkimux This is helpful to see--I'm working on a review of the alert component as the design system team works on migration to USWDS v3, and included the note that while our current alert uses H2, that can cause an incorrect heading hierarchy, and recommended following USWDS v3's lead with having it be h4--less likely to be too high a heading level, and less of a problem if a level is skipped.

**Comment by jaredcunha** [comment]
@jstrothman Is it possible to use a prop like `headingLevel` to ensure the heading is always semantically correct? USWDS alert headings can be set to any level, but they have to pick one for their examples. The CSS class sets the font-size, so it can anything from h1-h6 without changing visually (I do think they should call that out in their guidance).

**Comment by jstrothman** [comment]
@jaredcunha @joshkimux I moved too quickly--was just looking through the storybook for the guidance about heading level and see that it already supports changing the heading level, like you said. I'll revise my notes accordingly, including explicitly stating that the heading level can be changed without changing anything visually.

**Comment by jaredcunha** [comment]
@jstrothman @joshkimux I've wrote an [issue](https://github.com/uswds/uswds/issues/5260) for the USWDS, because I've been hardcoded headings in other projects.

Ah - I didn't notice that the [headline guidance](https://design.va.gov/storybook/?path=/docs/components-va-alert--heading-level) in Storybook at first. I was looking at the `props` list, which I assumed the component was more tightly scoped, and didn't require the full HTML for the heading.
